### ISSUE TITLE
Build instructions

### ISSUE THREAD

**Issue by leoheck** [thliebig/AppCSXCAD#5]
I am trying to build this repo, since the APP is not working for me on Ubuntu 20.04

The [INSTALL](https://github.com/thliebig/AppCSXCAD/blob/master/INSTALL) file, line 9, has this info

```
  - change directory to QCSXCAD
```

Is it right, o it should say AppCSXCAD?

**Comment by thliebig** [comment]
Yes that looks like a copy paste error...
But I recommend to build everything at once using the [openEMS-Project](https://github.com/thliebig/openEMS-Project)?

**Comment by leoheck** [comment]
Cool, thanks. I can submit a fix if you agree.


So, my history is this... 

I installed openEMS from the OS package manager. I think it has the AppCSXCAD and works just fine out of the box.

Then later I was trying to execute this [pcbmodelgen](https://github.com/jcyrax/pcbmodelgen) application that converts Kicad boards to analyze with openEMS. So, I was having a bad time with that. And then I tried to build the openEMS manually. 

This manual build messed with my previous installation. Then I was making progress fixing some issues in the build system, but I was not able to build the `AppCSXCAD`. This is why I was investigating here.

Long story short, I was able to fix my installation a couple of hours ago. The issue was that the old/wrong/different version of the `libCSXCAD.so` was standing in front of the right one.

My fix was just a matter of adjusting the symbolic link of the library that was being used for the openEMS/AppCSXCAD
```
➜ sudo ln -sf /usr/lib/x86_64-linux-gnu/libCSXCAD.so.0 /usr/local/lib/libCSXCAD.so.0
```

**Comment by biergaizi** [comment]
The latest manual build instructions can be found at https://openems.readthedocs.io/en/latest/development/cpp.html We probably should deprecate the content of `INSTALL` and points users to `openEMS-Project`'s documentation instead. I'll do that in a Pull Request to close this bug.
